GameDefinition/Content/ItemKind.hs

```diff
@@ -607,7 +607,7 @@ potion8 = potionTemplate
                , toOrganGood "rose-smelling" (80 + 1 `d` 20)
                , OnSmash (Explode "healing mist 2")
                , OnSmash (Explode "distressing odor") ]
-  -- , idesc    = ""
+  , idesc    = "Swirls of intense colour draw your eyes."
```

Comment:
Heh, I was aiming at a romance drama with this one (a unique artifact `potion of Love`): a full heal effect, but equally huge distress (and an outward sweet smell appearance with a hidden dramatic pangs of odour of rejection, I guess).

Reply:
This one doesn't draw my eyes as much as the others. Apart of the silly romantic theme, the potion presents a choice (in some situations): either heal friend for lots of HP (the altruistic side of romantic love) or dominate a foe who is already impressed with us (the `Impress` effect) by bringing his Calm to zero (the enslaving side of romantic love) and then he fights for us. I guess I invested too much silliness into that one, so I won't be easily satisfied. :)